Algorithm.CSharp/CustomPartialFillModelAlgorithm.cs

```diff
@@ -89,20 +90,71 @@ public override OrderEvent MarketFill(Security asset, MarketOrder order)
                 // Set this fill amount
                 fill.FillQuantity = Math.Sign(order.Quantity) * 10;
```

Comment:
We could take this chance here and get the max of this quantity and te signed remaining one, for cases where the original quantity is not a multiple of 10. Just to make the example more accurate.
Same for Python version.